src/internal/api/get_status_handler.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+type getStatusResponse struct {
+	Chain  string `json:"chain"`
+	Status string `json:"status"`
+}
+
+func getStatusHandler(s *Server, _ *fiber.Ctx) (interface{}, int, error) {
```

Comment:
Why is it necessary to add `*fiber.ctx` as paramater of the function?

Reply:
to comply with `type handler func(*Server, *fiber.Ctx) (interface{}, int, error)` definition

Reply:
And why does that definition uses fiber as param, is the unique handler that you have doesn't need it?

Reply:
because we are going yo have more handlers in the furure

Reply:
ok 👍